Dockerfile

```diff
@@ -12,7 +12,7 @@ LABEL maintainer="Rhys Arkins <rhys@arkins.net>" \
   org.opencontainers.image.source="https://github.com/containerbase/buildpack"
 
 #  autoloading buildpack env
-ENV BASH_ENV=/usr/local/etc/env ENV=/usr/local/etc/env PATH=/home/user/bin:$PATH
+ENV BASH_ENV=/usr/local/etc/env ENV=/usr/local/etc/env PATH=/opt/buildpack/bin:/home/user/bin:$PATH
```

Comment:
@rarkins should we place the symlinks and shell wrapper to `/use/local/bin`, so we don't have to change the `PATH` ?

Reply:
I would prefer not to do that, so that we have a clean separation.
This would be a breaking change but we can do that together with all tools for v3.

Reply:
I like to use an existing path folder, as we had and have trouble with it.

Some container runtimes / ci tools override path, so our buildpack tools won't be found.
I've seen this especially on Jenkins.

@rarkins WDYT?

Reply:
So we will set the default bin path to `/usr/local/bin`?
I can make the changes for all v2 functions.

Reply:
I would very much like to use symlinks and not be dynamically switching path

Reply:
👍 I will change the bin path from `/opt/buildpack/bin` to `/usr/local/bin` in a separate PR

Reply:
Ok, we should merge that before this.

Reply:
BTW I'm not wedded to `/usr/local/bin`.. it can be another directory like `/opt/buildpack/bin`. For me the most important part is that the `PATH` doesn't need to keep switching

Reply:
```suggestion
ENV BASH_ENV=/usr/local/etc/env ENV=/usr/local/etc/env PATH=/home/user/bin:$PATH
```